frontend/src/components/ConfigGeneratedJson.vue

```diff
@@ -1,12 +1,14 @@
 <template>
   <v-container>
-    <v-textarea
-      auto-grow
-      readonly
-      label="Application Configuration"
-      v-model="appConfigAsString"
-      class="jsonText"
-    ></v-textarea>
+    <div class="">
```

Comment:
Does class need to be specified when it's empty?

Reply:
No